bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/SpecialCharacterConstants.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2023 ETAS GmbH and others, all rights reserved.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ETAS GmbH - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.jface.text;
+
+/**
+ * @since 3.25
+ */
+public class SpecialCharacterConstants {
```

Comment:
```suggestion
class SpecialCharacterConstants {
```

should work as well....